Arduino/Ground Systems/Controls/controls.ino

```diff
@@ -44,7 +44,7 @@ void loop()
       {
         for (int j = 1; j < i - 1; j++)
         {
-          int pin = pins[j - 1];
+          // int pin = pins[j - 1];
```

Comment:
I'm in favor of just removing the line altogether. If we need it in the future, we can utilize version control to get it back.